src/main/php/de/thekid/dialog/processing/Videos.php

```diff
@@ -18,6 +20,26 @@ private function execute(string $command, array<string> $args): void {
   }
 
   public function meta(File $source): array<string, mixed> {
+    if (preg_match('/\.(mov|mp4|mpeg)$/i', $source->getFileName())) {
+      $meta= [];
+      foreach ($this->atoms->in($source) as $name => $atom) {
+        if ('moov.meta.keys' === $name) {
+          $meta= $atom['value'];
+        } else if ('moov.meta.ilst' === $name) {
+          $meta= array_combine($meta, $atom['value']);
+        }
+      }
+
+      // FIXME: This needs to support more than just Apple!
+      if (isset($meta['mdta:com.apple.quicktime.software'])) {
```

Comment:
We need some videos from an Android phone

Reply:
The `mvhd` atom contains this information in form of the *Creation time* and *Modificatio time* fields represented by unsigned ints since January 1st, 1904 midnight UTC, see https://developer.apple.com/documentation/quicktime-file-format/movie_header_atom